scripts/known_good.json

```diff
@@ -17,7 +17,7 @@
       "sub_dir" : "Vulkan-Headers",
       "build_dir" : "Vulkan-Headers/build",
       "install_dir" : "Vulkan-Headers/build/install",
-      "commit" : "c4e056d365472174471a243dfefbfe66a03564af"
+      "commit" : "db09f95ac00e44149f3894bf82c918e58277cfdb"
```

Comment:
I believe the Loader commit ID should be updated as well. I'm not sure about the glslang commit ID--for VVL I update it in the general scripts directory, but not for the Android known-good.

Reply:
Cool, I went ahead and did them all.